extra/git-vcs.vim

```diff
@@ -1,19 +1,24 @@
 " Git checkout
+let g:WhichKeyDesc_GitVcs_Checkout = "<leader>gb checkout"
```

Comment:
```suggestion
let g:WhichKeyDesc_GitVcs_Branches = "<leader>gb branches"
```

I know the action is a checkout, but `b` stands for "branches" 😅
What do you think?
Maybe we call it `branch-checkout`?

Reply:
Good catch.  TBH, not sure why I named it Checkout......let's stick with your suggested change.  I'll commit it.